pyproject.toml

```diff
@@ -60,6 +60,7 @@ sphinx_togglebutton = "^0.3.2"
 sphinx-favicon = "1.0.1"
 sphinx-autoapi = "^3.0.0"
 kaleido = "0.2.1"
+poetry-dotenv-plugin = "^0.2.0"
```

Comment:
This loads variables from `.env` into the poetry shell or `poetry run` commands.